README.md

```diff
@@ -1,17 +1,16 @@
 # ML Masterclass
 
-This is the course repo for the Masterclass.
+This is the course repo for the AI Masterclass.
 
-The structure of the repo consists of:
+The structure of the repo consists of a directory for each section. Each section contains multiple directories for the hands-on projects:
 
-* 0_introduction_to_tensorflow (Week 1)
-* 1_model_deployment (Week 1)
-* 2_large_language_models (Weeks 4 & 5)
-* 3_computer_vision (Weeks 6 & 7)
+* `0_introduction_to_tensorflow` (Week 1)
+* `1_model_deployment` (Week 1)
+* `2_large_language_models` (Weeks 4 & 5)
+* `3_computer_vision` (Weeks 6 & 7)
```

Comment:
You can actually remove the week information from here. Let them stay as 4 separate modules numbered from 0 to 3..